src/wiktextract/extractor/es/pronunciation.py

```diff
@@ -88,8 +93,17 @@ def process_pron_graf_template(
             if len(spelling_v.model_dump(exclude_defaults=True)) > 1:
                 spelling_data.append(spelling_v)
 
-        if len(sound.model_dump(exclude_defaults=True)) > 0:
-            sound_data.append(sound)
+        main_sound = variations[0]
+        for key in main_sound.model_fields_set:
+            for i, other_variation in variations.items():
+                if i == 0:
+                    continue
+                if key not in other_variation.model_fields_set:
+                    other_variation.__setattr__(key, 
+                                main_sound.__getattribute__(key))
```

Comment:
Should use `setattr` and `getattr`.

Reply:
Yeah, already switched when I saw those in some other place. I'm finding a ton of issues (and trying to fix them), now that I actually tried to test this, so no need currently to check this yet...